### ISSUE TITLE
Have develop command pick next randomly available port

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/greenwood#71]
## Type of Change
- [x] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [ ] Bug
- [ ] Other (please clarify below)

## Summary
By default, instead of hardcoding the port for `devServer` it might be really nice to have it just pick the next available port on the user's local machine?

## Details

Something similar to  [**webpack-dev-server**](https://stackoverflow.com/questions/42055201/webpack-dev-server-dynamic-port)

> _Also, if we enable this, we might / should be able to run all our tests in parallel?  Doesn't have to be in this PR, but a good fast follow if so!_

**Comment by thescientist13** [comment]
I think we can probably get that output as part of running **webpack-dev-server** maybe? 

Also, probably for a related issue, would be adding `open` as default?

**Comment by hutchgrant** [comment]
To my knowledge, that output doesn't show by default. We're showing it on purpose after build because otherwise user would have no idea. `open` didn't work in my OS, under every test I ran with it.  Yet, it does work in CRA so I don't know.

**Comment by thescientist13** [comment]
ah, interesting.  Maybe in most other projects I'm always using the CLI, so always passed the flag that way
```
$ webpack-dev-server --open
```

**Comment by hutchgrant** [comment]
yea might be the difference between webpack node api and cli. There are small things such as that which are different.

**Comment by thescientist13** [comment]
I wonder if [this trick](https://twitter.com/matteocollina/status/1255888871684014080) would work?
>  would not hardcode the port and set it to zero so it is dynamically allocated, and then start the client and communicate to it. It's the only sane way I've found to avoid flaky tests.
